ios/Podfile.lock

```diff
@@ -777,10 +778,35 @@ PODS:
     - React-Core
   - RNReactNativeHapticFeedback (1.14.0):
     - React-Core
-  - RNReanimated (3.6.1):
-    - RCT-Folly (= 2021.07.22.00)
+  - RNReanimated (3.5.4):
```

Comment:
@arosiclair Is there a reason why Reanimated is downgraded here?

Reply:
I'm not sure exactly. When I pulled `main` and  pod installed, this change appeared. I'll try again and see what happens

Reply:
Fixed with another `pull` and `pod install`